LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl

```diff
@@ -561,20 +610,30 @@
 
         // get data for any custom fields
         let getData = [];
-        if (site.columns != undefined && Array.isArray(site.columns.split(','))) {
-            let cols = site.columns.split(',');
-            for (let i in site.columns.split(',')) {
-                if (!isNaN(parseInt(cols[i]))) {
-                    getData.push(parseInt(cols[i]));
-                } else {
-                    switch (cols[i].toLowerCase()) {
-                        case 'days_since_last_action':
-                        case 'email_reminder':
-                            query.join('action_history');
-                            break;
-                        default:
-                            break;
-                    }
+
+        const siteColumns = (site?.columns || "").split(',').filter(c => c !== "");
+        const formColumns = site?.formColumns || {};
+        let arrFormColumns = [];
+        for (let key in formColumns) {
+            let cols = formColumns[key];
```

Comment:
Variables shouldn't be defined inside of loops.  The definition  should be set outside, and then use a conditional assignment so it either puts in the new value or null.